Backpack/src/main/java/net/skyscanner/backpack/map/internal/BpkMapMarkersTestBridge.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ * <p>
+ * Copyright 2018-2021 Skyscanner Ltd
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.map.internal;
+
+import android.content.Context;
+import android.view.View;
+
+// This class is used for screenshots testing only.
+// It's not a part of public API and may be changed at any moment without being considered as a breaking change.
+import org.jetbrains.annotations.NotNull;
+
+/*package*/ enum BpkMapMarkersTestBridge {
+  Instance;
+
+  @NotNull
+  /*package*/ View create(@NotNull Context context, @NotNull String title, int icon, boolean showPointer) {
+    return BpkMapViewsKt.createBpkMarkerView(context, title, icon, showPointer);
```

Comment:
As we declare createBpkMarkerView with internal visibility modifier, it cannot be used outside its module. However our tests are located in a different module(app).

To bypass it, we use Java singleton (as enum) with package visibility modifier. In this case any module can use this code providing the code has the same package.

To prevent any other usages we have the very special comment :)